package/metadata.desktop

```diff
@@ -4,20 +4,28 @@ Comment=Run a command periodically and render its output.
 
 Type=Service
 Icon=utilities-terminal
-X-KDE-ServiceTypes=Plasma/Applet
-
+KPackageStructure=Plasma/Applet
 X-Plasma-API=declarativeappletscript
-X-Plasma-MainScript=ui/main.qml
+
+X-Plasma-API-Minimum-Version=6.0
 
 X-KDE-PluginInfo-Author=Chris Holland
 X-KDE-PluginInfo-Email=zrenfire@gmail.com
+X-KDE-PluginInfo-Author=Ari Caldeira
+X-KDE-PluginInfo-Email=aricaldeira@gmail.com
+X-KDE-PluginInfo-Author=Oded Arbel
+X-KDE-PluginInfo-Email=oded@geek.co.il
```

Comment:
Multiple `X-KDE-PluginInfo-Author` works?! Does using `desktoptojson` create multiple entries?

I need to rewrite my build/install scripts to support `metadata.json`. I did sorta write one in python a while ago which I should revisit. I think I got stuck at merging the i18n back into the `metadata.json` and realized it was simplier to keep the desktop.

The problem is that Plasma6 doesn't like it when the `.desktop` file exists.

Reply:
seems like it works, you can see the output from
```sh
kpackagetool6 --appstream-metainfo package/
```

```xml
<?xml version="1.0" encoding="UTF-8"?>
<component>
    <id>com.github.aricaldeira.commandoutput</id>
    <name>Command Output</name>
    <name xml:lang="bz">Saida di Komandus</name>
    <name xml:lang="fr">Command Output</name>
    <name xml:lang="nl">Opdrachtuitvoer</name>
    <name xml:lang="pt_br">Saída de Comandos</name>
    <summary>Run a command periodically and render its output</summary>
    <summary xml:lang="bz">Ezekuta un komandu a kada X milisegundus, i mòstra a saida/rezuwtadu</summary>
    <summary xml:lang="fr">Exécute une commande périodiquement et affiche son retour</summary>
    <summary xml:lang="nl">Voer een opdracht periodiek uit en toon de uitvoer ervan</summary>
    <summary xml:lang="pt_br">Executa um comando a cada X milissegundos, e mostra a saída/resultado</summary>
    <url type="homepage">https://github.com/aricaldeira/plasma-applet-commandoutput</url>
    <developer id="kde.org">
        <name>Chris Holland, Ari Caldeira, Oded Arbel</name>
    </developer>
    <icon type="stock">utilities-terminal</icon>
    <project_license>GPL-2.0+</project_license>
    <metadata_license>CC0-1.0</metadata_license>
</component>
```